### PR TITLE
Detect pipeline attribute during compile/run

### PR DESCRIPTION
**Pull request by PhilippeMoussalli**
Addresses #375

### CODE REVIEW

**Comment by GeorgesLorre** [comment]
Nice change ! Makes it easier the import string was indeed a bit confusing.

Some notes:
1. did you check the documentation for cli examples that might need updating ?
2. In the original implementation when there was an error in you pipeline code the cli would just say that it could not find your pipeline but there was never any indication (traceback or special message) that there was something wrong is this easier in this implementation ?

**Comment by PhilippeMoussalli** [comment]
> Nice change ! Makes it easier the import string was indeed a bit confusing.
> 
> Some notes:
> 
> 1. did you check the documentation for cli examples that might need updating ?
> 2. In the original implementation when there was an error in you pipeline code the cli would just say that it could not find your pipeline. There was never any indication (traceback or special message) that there was something wrong, is this easier in this implementation ?

Thanks Georges! 

1) Just update the docs. 
2) If there is no pipeline found it will return a valid error. I think the error that you're referring to was because of the default output path (which was wrongly set to None). This shouldn't occur anymore. It's either the default for kfp/docker-compose or a user custom one.

**Review by GeorgesLorre — APPROVED** [review]

File: docs/getting_started.md [hunk 1]
```diff
@@ -287,7 +287,7 @@ We add the component to our pipeline definition and specify that it depends on t
 We can now easily run our new pipeline:
 
 ```bash
-fondant run pipeline:my_pipeline --local
+fondant run pipeline --local
```
Comment (mrchtr) [inline docs/getting_started.md:290]: I think one command is outdated here. Either the one in line 142 `fondant run pipeline.py --local` or this one.

File: docs/getting_started.md [hunk 2]
```diff
@@ -139,10 +139,10 @@ Fondant has a feature rich CLI that helps you with these steps. Let's start by r
 First of all make sure you have [Docker Compose](https://docs.docker.com/compose/) installed on your system
 
 ```bash
-fondant run pipeline:my_pipeline --local
+fondant run pipeline.py --local
```
Comment (mrchtr) [inline docs/getting_started.md:142]: Shouldn't this be `fondant compile pipeline --local`? Is the `.py` needed here?

**Review by mrchtr — APPROVED** [review]
Thanks @PhilippeMoussalli, lgtm. I've just found two small typos in the documentation.

**Pull request merged**